From 3a8a267b671e7dac6123d3eec6fe6c19a8b3cd90 Mon Sep 17 00:00:00 2001
From: Heinrich Toews <heinrich.toews@wago.com>
Date: Thu, 17 Jan 2019 11:55:58 +0100
Subject: [PATCH] dts: imx6qdl-vtpctp: remove rs485-rts-delay property

The imx serial driver does not support rs485-rts-delay configuration.

Signed-off-by: Heinrich Toews <heinrich.toews@wago.com>
---
 arch/arm/boot/dts/imx6qdl-vtpctp.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx6qdl-vtpctp.dtsi b/arch/arm/boot/dts/imx6qdl-vtpctp.dtsi
index 4280f6b..b108659 100644
--- a/arch/arm/boot/dts/imx6qdl-vtpctp.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-vtpctp.dtsi
@@ -1041,7 +1041,6 @@
 	rs485-rx-during-tx;
 	rts-gpio = <&gpio3 29 GPIO_ACTIVE_HIGH>;
 	linux,rs485-enabled-at-boot-time;
-	rs485-rts-delay = <0 0>;
 };
 
 &usbh1 {
-- 
2.7.4

